src/aleph/vm/utils.py

```diff
@@ -218,3 +218,8 @@ def file_hashes_differ(source: Path, destination: Path, checksum: Callable[[Path
         return True
 
     return checksum(source) != checksum(destination)
+
+
+def write_bytes_file(file_path: Path, file_content: bytes):
+    """Save a file on disk in async way."""
+    file_path.write_bytes(file_content)
```

Comment:
I do not understand this function 🤨
1. How is it better to use `write_bytes_file(path, bytes)` instead of `path.write_bytes(bytes)` ?
2. How is this asynchronous ? Should [aiofiles](https://pypi.org/project/aiofiles/) be used instead ?

Reply:
Initially I tried to write files in async way, and I created that method for that thing. But after I get that the files received by the endpoint are in memory and are so small, so isn't needed to write the files in async way. So finally I made it sync and I forgot to remove that method. Now I have removed it.